types/modules/producer.d.ts

```diff
@@ -10,6 +10,8 @@ declare class ProducerError extends Error {
 interface ProduceOptions extends amqp.Options.Publish {
   routingKey?: string;
   rpc?: boolean;
+  /** Timeout in milliseconds for producing RPC request and waiting for the response. Does not affect non-rpc requests. */
```

Comment:
```suggestion
  // Timeout in milliseconds for producing RPC request and waiting for the response. Does not affect non-rpc requests.
```

Reply:
Why comment needed, specifically for this field?
I suggest either document all properties, or none. Lets be consistent.

Reply:
The options are not intuitive, so I added docs to all of them.